src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarItem.cs

```diff
@@ -21,38 +18,27 @@ internal abstract class NavigationBarItem
         public int Indent { get; }
         public ImmutableArray<NavigationBarItem> ChildItems { get; }
 
-        public ImmutableArray<TextSpan> Spans { get; internal set; }
-        internal ImmutableArray<ITrackingSpan> TrackingSpans { get; set; } = ImmutableArray<ITrackingSpan>.Empty;
```

Comment:
no need for 'Spans' anymore in the editor api.  nothing uses that.  ANd TRackingSpans is no longer mutable as it is provided at construction time.